From e31ed854a1a5056f5f70bce49bf9daa4566b428f Mon Sep 17 00:00:00 2001
From: Dan Lambright <dlambrig@redhat.com>
Date: Mon, 19 Oct 2015 14:16:42 -0400
Subject: [PATCH 40/67] cluster/tier update man pages for tier feature

Add to gluster man pages instructions for tier commands.

Backport fix 12391

> Change-Id: I0918460eeaba22bb6a11238d4f5501fa8e61da88
> BUG: 1272557
> Signed-off-by: Dan Lambright <dlambrig@redhat.com>
> Reviewed-on: http://review.gluster.org/12380
> Tested-by: NetBSD Build System <jenkins@build.gluster.org>
> Reviewed-by: N Balachandran <nbalacha@redhat.com>

Change-Id: I2940b5a77207e5ebf8d743ce862a05bf2c77e8a1
BUG: 1272403
Signed-off-by: Dan Lambright <dlambrig@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/59793
---
 doc/gluster.8 |   19 +++++++++++++++++++
 1 files changed, 19 insertions(+), 0 deletions(-)

diff --git a/doc/gluster.8 b/doc/gluster.8
index e1e2a60..c09c1ee 100644
--- a/doc/gluster.8
+++ b/doc/gluster.8
@@ -105,6 +105,25 @@ Display the status of peers.
 .TP
 \fB\ peer help \fR
 Display help for the peer command.
+.SS "Tier Commands"
+.TP
+\fB\ volume tier <VOLNAME> attach [<replica COUNT>] <NEW-BRICK>... \fR
+Attach to an existing volume a tier of specified type using the specified bricks.
+.TP
+\fB\ volume tier <VOLNAME> status \fR
+Display statistics on data migration between the hot and cold tiers.
+.TP
+\fB\ volume tier <VOLNAME> detach start\fR
+Begin detaching the hot tier from the volume. Data will be moved from the hot tier to the cold tier.
+.TP
+\fB\ volume tier <VOLNAME> detach commit [force]\fR
+Commit detaching the hot tier from the volume. The volume will revert to its original state before the hot tier was attached.
+.TP
+\fB\ volume tier <VOLNAME> detach status\fR
+Check status of data movement from the hot to cold tier.
+.TP
+\fB\ volume tier <VOLNAME> detach stop\fR
+Stop detaching the hot tier from the volume.
 .SS "Bitrot Commands"
 .TP
 \fB\ volume bitrot <VOLNAME> {enable|disable} \fR
-- 
1.7.1

